lexicons/com/atproto/resource/account.json

```diff
@@ -0,0 +1,40 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.resource.account",
```

Comment:
I'm not sure about having these all in the same "resource". We're kinda mixing together the scariest permissions (deleteAccount) with the permissions that basically every app will be requesting access to (viewEmail).

An alternative would be to split this into several resources such as:
- com.atproto.resource.email
- com.atproto.resource.account
- com.atproto.resource.app_password

Reply:
+1 for splitting.

What do you think about:

- `com.atproto.account.deletion`
- `com.atproto.account.activation` // No params (allows to activate/deactivate)
- `com.atproto.account.email?view=true&update=false`
- `com.atproto.account.appPassword?create=false&revoke=true&list=true`

Reply:
yeah I dig that quite a bit. I think it's easier to reason about & prevents long possibly confusing query strings